.gitlab/pipeline/test.yml

```diff
@@ -68,7 +69,7 @@ test-linux-stable-runtime-benchmarks:
     # but still want to have debug assertions.
     RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
   script:
-    - time cargo nextest run --workspace --features runtime-benchmarks benchmark --locked --cargo-profile testnet
+    - time cargo nextest run --filter-expr 'not deps(/polkadot-subsystem-bench/)' --workspace --features runtime-benchmarks benchmark --locked --cargo-profile testnet
```

Comment:
Why isn't this `required-features = ["subsystem-benchmarks"]` enough ?

Reply:
It's a good question, and I wish I knew the answer. Somehow, it keeps running regression tests while `cargo test` exclude them based on required feature.